app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt

```diff
@@ -105,7 +115,11 @@ class MessageContentMapper @Inject constructor(
     private suspend fun mapRegularMessage(
         message: Message.Regular,
     ) = when (val content = message.content) {
-        is Asset -> toAsset(conversationId = message.conversationId, messageId = message.id, assetContent = content.value)
+        is Asset -> {
+            val assetMessageData = AssetMessageData(content.value)
+            val assetMessage = toUIMessageContent(assetMessageData, message, scope)
+            assetMessage
```

Comment:
Is this `assetMessage` required or can we just return `toUIMessageContent(assetMessageData, message, scope)`?